cdap-ui/app/cdap/components/AbstractWidget/FormInputs/TextBox/index.tsx

```diff
@@ -23,6 +24,7 @@ import { objectQuery } from 'services/helpers';
 interface ITextBoxWidgetProps {
   placeholder?: string;
   default?: string;
+  enableUnderline?: boolean;
```

Comment:
This should probably get added to the widget attributes as well.

Reply:
I'd prefer to address that for other widgets in another PR. In particular, I'd prefer to have this be `disableUnderline` as the material-ui components have it, but the default is already set for our widgets. Maybe it can be addressed during the UI framework work.

Reply:
Got it. In that case, maybe we can go ahead and create a JIRA for that (or document it elsewhere) and add a TO DO comment?

Reply:
Added https://issues.cask.co/browse/CDAP-17244